indico/core/signals/event/designer.py

```diff
@@ -11,5 +11,7 @@
 _signals = Namespace()
 
 print_badge_template = _signals.signal('print-badge-template', """
-Called when printing a badge template. The registration form is passed in the `regform` kwarg.
+Called when printing a badge template. 
+The registration form is passed in the `regform` kwarg.
+The registration objects are passed in the `registrations` kwargs
```

Comment:
`kwarg` instead of `kwargs`

Reply:
`kwarg` typo fixed

Reply:
I think you changed it the wrong way round ;) it should be "kwarg", not "kwargs" (since each sentence references a single kwarg).

Reply:
Alright